### PR TITLE
Update user guide to include `interview` and `remove-i` commands

### PR DESCRIPTION
**Pull request by jhchee18**
Update user guide to include `interview` and `remove-i` commands.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -245,6 +289,8 @@ We will assist you as soon as possible.
 | **Delete**      | `delete INDEX`<br> e.g., `delete 2`                                                                                                                                                         |
 | **Find**        | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find engineer`                                                                                                                                    |
 | **Edit**        | `edit INDEX [n/COMPANY] [ct/CONTACT] [e/EMAIL] [p/POSITION] [d/DATE_APPLIED] [t/TAG]...`<br> e.g., `edit 2 c/Grab d/2022-10-10`                                                             |
+| **Interview**   | `interview INDEX ir/ROUND id/INTERVIEW_DATE it/INTERVIEW_TIME il/LOCATION`<br> e.g., `interview 5 ir/Technical interview id/2024-09-16 it/1400 il/11, Kallang Way 2, #08-15, 119546`        |
+| **Remove-i**    | `remove-i`<br/> e.g., `remove-i 3`                                                                                                                                                          |
```
Comment (jolynloh) [inline docs/UserGuide.md]:
```suggestion
| **Remove-i**    | `remove-i INDEX`<br/> e.g., `remove-i 3`                                                                                                                                                          |
```

**Review by jolynloh — COMMENTED** [review]
LGTM!

File: docs/UserGuide.md (hunk 1 above)
Comment (jhchee18) [reply docs/UserGuide.md]: Thanks for spotting that! 😃

**Review by nmtuan2001 — COMMENTED** [review]
LGTM! Just one thing, I was thinking if it would be a good idea to change `remove-i` to `delete-i`? No huge difference but just for consistency (since for application, the command is `delete`).

**Comment by jhchee18** [comment]
> LGTM! Just one thing, I was thinking if it would be a good idea to change `remove-i` to `delete-i`? No huge difference but just for consistency (since for application, the command is `delete`).

The idea of using `remove` instead of `delete` is because the concept of "taking the interview away from the application". Another thing is I think using remove-i the user is easier to differentiate between them? So not that easy to get confused.

**Comment by nmtuan2001** [comment]
> The idea of using `remove` instead of `delete` is because the concept of "taking the interview away from the application". Another thing is I think using remove-i the user is easier to differentiate between them? So not that easy to get confused.

Hmm yeah that's a good point. I was thinking that having the command word `remove-i` in a `DeleteInterviewCommand` class feels a little inconsistent. Do you think naming the class as `RemoveInterviewCommand` would be better? If not, it's fine coz this is really very minor.

**Comment by jhchee18** [comment]
> > The idea of using `remove` instead of `delete` is because the concept of "taking the interview away from the application". Another thing is I think using remove-i the user is easier to differentiate between them? So not that easy to get confused.
> 
> Hmm yeah that's a good point. I was thinking that having the command word `remove-i` in a `DeleteInterviewCommand` class feels a little inconsistent. Do you think naming the class as `RemoveInterviewCommand` would be better? If not, it's fine coz this is really very minor.

Yea, make sense. Ok, I am changing the class to `RemoveInterviewCommand`.

**Pull request merged**